tests/Unit/RequestTest.php

```diff
@@ -399,6 +399,23 @@ public function test_skip_cache()
         $this->assertEquals('true', $api->getQuery()['skipCache']);
     }
 
+    public function test_set_route_with_query_string()
+    {
+        $api = Request::local()->setRoute('http://local.test/?page=1&foo=bar');
+
+        $query = $api->getQuery();
+
+        $this->assertArrayHasKey('page', $query);
+        $this->assertArrayHasKey('foo', $query);
```

Comment:
Adiciona tbm `assertEquals` pros 2 casos, pra validar que o valor correto está sendo adicionado
```suggestion
        $this->assertArrayHasKey('page', $query);
        $this->assertArrayHasKey('foo', $query);
        $this->assertEquals('1', $query['page']);
        $this->assertEquals('bar', $query['foo']);
```